shop-be/product-service/serverless.ts

```diff
@@ -1,7 +1,10 @@
 import type { AWS } from '@serverless/typescript';
+require('dotenv').config()
```

Comment:
For lambdas all this variables usually passed via environment variables in AWS lambda settings.
The value can be stored in SSM or Secrets Manager -https://www.serverless.com/framework/docs/providers/aws/guide/variables#reference-variables-using-the-ssm-parameter-store.
It allows do not store any local files and keep all secrets in AWS.